onecodex/viz/_heatmap.py

```diff
@@ -145,6 +145,7 @@ def plot_heatmap(
         tooltip.insert(0, "Label")
 
         magic_metadata, magic_fields = self._metadata_fetch(tooltip, label=label)
+        magic_metadata.replace(np.nan, "None", inplace=True)
```

Comment:
Can we use `"N/A"` to match the other places where we display missing values?

Also a unit test would be nice

Reply:
Updated